velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1114,6 +1327,42 @@ void JoinFuzzer::addPlansWithTableScan(
         groupedProbeScanSplits,
         groupedBuildScanSplits));
   }
+
+  // Add ungrouped MergeJoin with TableScan.
+  if (joinNode->isInnerJoin() || joinNode->isLeftJoin()) {
+    makeMergeJoinPlanWithTableScan(
+        joinType,
+        probeType,
+        buildType,
+        probeKeys,
+        buildKeys,
+        probeScanSplits,
+        buildScanSplits,
+        outputColumns);
```

Comment:
Do we want to flip the build/probe sides for merge join?

Reply:
No, merge join only supports inner and left join. I've tried flipping but that's not allowed.